src/code/qualification/plotQualificationTimeProfile.m

```diff
@@ -265,50 +265,57 @@ function plotQualificationTimeProfile(WSettings,figureHandle,TimeProfile,Observe
     if strcmp(Curves.Statistics(i).Id, 'ArithmeticMean')
         p_handle(i) = plot(time, mean(Y));
-        legendLabels{i}=sprintf('Arithmetic Mean %s', Curves.Y);
+        legendLabels{i}=sprintf('%s-Arithmetic Mean', Curves.Y);
         setCurveOptions(p_handle(i), Curves.Statistics(i));
     end
     if strcmp(Curves.Statistics(i).Id, 'ArithmeticStandardDeviation')
         p_handle(i) = plot([time NaN time], [mean(Y)-std(Y) NaN mean(Y)+std(Y)]);
-        legendLabels{i}=sprintf('Arithmetic Standard Deviation %s', Curves.Y);
+        legendLabels{i}=sprintf('%s-Arithmetic Standard Deviation', Curves.Y);
         setCurveOptions(p_handle(i), Curves.Statistics(i));
     end
     if strcmp(Curves.Statistics(i).Id, 'GeometricMean')
         p_handle(i) = plot(time, geomean(Y));
-        legendLabels{i}=sprintf('Geometric Mean %s', Curves.Y);
+        legendLabels{i}=sprintf('%s-Geometric Mean', Curves.Y);
         setCurveOptions(p_handle(i), Curves.Statistics(i));
     end
     if strcmp(Curves.Statistics(i).Id, 'GeometricStandardDeviation')
         p_handle(i) = plot([time NaN time], [exp(mean(log(Y))-std(log(Y))) NaN exp(mean(log(Y))+std(log(Y)))]);
-        legendLabels{i}=sprintf('Geometric Standard Deviation %s', Curves.Y);
+        legendLabels{i}=sprintf('%s-Geometric Standard Deviation', Curves.Y);
         setCurveOptions(p_handle(i) , Curves.Statistics(i));
     end
     if strcmp(Curves.Statistics(i).Id, 'Median')
         p_handle(i) = plot(time, median(Y));
-        legendLabels{i}=sprintf('Median %s', Curves.Y);
+        legendLabels{i}=sprintf('%s-Median', Curves.Y);
         setCurveOptions(p_handle(i), Curves.Statistics(i));
     end
     if strcmp(Curves.Statistics(i).Id, 'Min')
         p_handle(i) = plot(time, min(Y));
-        legendLabels{i}=sprintf('Min %s', Curves.Y);
+        legendLabels{i}=sprintf('%s-Min', Curves.Y);
         setCurveOptions(p_handle(i), Curves.Statistics(i));
     end
     if strcmp(Curves.Statistics(i).Id, 'Max')
         p_handle(i) = plot(time, max(Y));
-        legendLabels{i}=sprintf('Max %s', Curves.Y);
+        legendLabels{i}=sprintf('%s-Max', Curves.Y);
         setCurveOptions(p_handle(i), Curves.Statistics(i));
     end
     if contains(Curves.Statistics(i).Id, 'Percentile')
         perc=sscanf(Curves.Statistics(i).Id, 'Percentile_%d')/100;
         p_handle(i) = plot(time, quantile(Y, perc));
-        legendLabels{i}=sprintf('Percentile %d %s ', perc*100, Curves.Y);
+        legendLabels{i}=sprintf('%s-Percentile %d%%', Curves.Y, perc*100);
         setCurveOptions(p_handle(i), Curves.Statistics(i));
     end
     if contains(Curves.Statistics(i).Id, 'Range')
         perc = sscanf(Curves.Statistics(i).Id, 'Range%d')/100;
         ran = quantile(Y, [(1-perc)/2 (1+perc)/2]);
+        if strcmpi(Scaling, 'Log')
+            % Remove 0s from plot in log
+            zeros2remove = min(ran==0);
+            time(zeros2remove) = [];
+            ran(:,zeros2remove) = [];
+        end

```

Comment:
@pchelle This is the fix for not-displayed range in log scale. Can you tell me where the other case (Mean curve hidden by the range in lin scale) is fixed?

Reply:
Yes, the other issue was that the range is not transparent and hide the mean curve.
I handled the issue by setting transparency line 319: set(p_handle(i), 'FaceAlpha', 0.5); 
The other solution would be to re-order the plots and always having the range before the other population time plots.